osbs/api.py

```diff
@@ -589,6 +589,18 @@ def get_image_stream_tag(self, tag_id):
         return self.os.get_image_stream_tag(tag_id)
 
     @osbsapi
+    def put_image_stream_tag(self, tag_id, tag):
```

Comment:
It's a building block. We could not expose it in `api.py`, and only leave it as a helper method in `core.py`. But I think it's generic enough that it could be used in the future for a different use case.